ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -58,11 +51,11 @@
 
     - name: Ensure we have needed bits for compute when needed
       when:
-        - has_compute | bool
+        - computes_len | int > 0
       ansible.builtin.assert:
         that:
-          - "'compute' in crc_ci_bootstrap_networks_out"
-          - "'default' in crc_ci_bootstrap_networks_out['compute']"
+          - "'compute-0' in crc_ci_bootstrap_networks_out"
```

Comment:
not sure we should still rely on that assertion.... @pablintino any thought? We'd be checking "only" on the first compute, not all?

Reply:
IIUC we need this assertion to get these default values for interface name and mtu:
`crc_ci_bootstrap_networks_out['compute-0'].default.mtu `
it is needed since it is still hardcoded below